zookeeper/src/jepsen/zookeeper.clj

```diff
@@ -112,21 +118,23 @@
          opts
          {:name    "zookeeper"
           :os      debian/os
-          :db      (db "3.4.9-3+deb9u1")
+          :db      (db "3.4.13-2")
           :client  (client nil nil)
           :nemesis (nemesis/partition-random-halves)
           :generator (->> (gen/mix [r w cas])
                           (gen/stagger 1)
                           (gen/nemesis
-                            (gen/seq (cycle [(gen/sleep 5)
-                                             {:type :info, :f :start}
-                                             (gen/sleep 5)
-                                             {:type :info, :f :stop}])))
+                           (cycle [(gen/sleep 5)
+                                   {:type :info, :f :start}
+                                   (gen/sleep 5)
+                                   {:type :info, :f :stop}]))
                           (gen/time-limit 15))
           :model   (model/cas-register 0)
           :checker (checker/compose
-                     {:perf   (checker/perf)
-                      :linear checker/linearizable})}))
+                    {:perf   (checker/perf)
+                     :linear (checker/linearizable 
+                              {:model (model/cas-register)
+                               :algorithm :linear})})}))
```

Comment:
It's commit to master branch, not master-yb.
Plan is to merge master with master-yb later since it's hard to work with outdated code. Also there is docker runner in Jepsen repo so we potentially can test it locally.